_shared/hooks/post_gen_project.py

```diff
@@ -24,6 +24,8 @@ def remove_conditional_files():
         "bin/create-db",
         "bin/make_db",
         "conf/alembic.ini",
+        "{{ cookiecutter.package_name }}/db.py",
+        "{{ cookiecutter.package_name }}/models/__init__.py",
```

Comment:
Don't add `db.py` and `models/` to projects that don't use the `postgres` option